common/src/main/java/com/emc/mongoose/common/concurrent/Daemon.java

```diff
@@ -22,7 +22,7 @@ void shutdown()
 	boolean isShutdown()
 	throws RemoteException;
 	
-	boolean await()
+	void await()
 	throws InterruptedException, RemoteException;
 
 	boolean await(final long timeout, final TimeUnit timeUnit)
```

Comment:
this await has only 2 exit variants (success, InterruptedException) while await(timeout, timeUnit) has 3 exit variants (success=true, timeout=false, InterruptedException)